android/vpnClient/build.gradle

```diff
@@ -67,6 +61,14 @@ android {
             includeAndroidResources = true
         }
     }
+
+    sourceSets {
+        main {
+            kotlin.srcDirs = [
+                "${System.getProperty('user.dir')}/../../clienttelemetry"
```

Comment:
Not a fan of this, can you either infer that from `$rootDir` or get that copied into the project?

Reply:
I can infer from `$rootDir`. But that would require me to assume the name of the build folder, which doesn't seem right. I also didn't want to generate these files outside of the build folder... I can copy them into the projec thought. Hadn't thought of that.

Reply:
> I can infer from $rootDir. But that would require me to assume the name of the build folder, which doesn't seem right.

Neverming me being wrong here... 🤦‍♀️